src/tools/idlpp/CMakeLists.txt

```diff
@@ -9,14 +9,9 @@
 #
 # SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
 #
-if(CMAKE_CROSSCOMPILING)
```

Comment:
Maybe you check for this one directory up (didn't feel like scrolling back :sweat_smile:), but shouldn't we check for `BUILD_IDLC`?